tests/vmcheck/test-autoupdate.sh

```diff
@@ -127,12 +127,12 @@ vm_start_httpd ostree_server $REMOTE_OSTREE 8888
 
 change_policy() {
   policy=$1; shift
-  vm_cmd cp /usr/etc/rpm-ostreed.conf /etc
-  cat > tmp.sh << EOF
-echo -e "[Daemon]\nAutomaticUpdatePolicy=$policy" > /etc/rpm-ostreed.conf
+  vm_ansible_inline <<EOF
+- shell: |
```

Comment:
Hmm, though we could also `vm_inline_shell`...tempting.

Reply:
Yep, that makes sense to me!

I suppose also some of the things we use shell for already have an ansible interface. E.g. here `lineinfile`, the `user` module for the `useradd` below, etc...